src/cfn.py

```diff
@@ -142,41 +142,25 @@ def helpfn(altkey):
 
 @task
 @requires_project
-def launch(pname, instance_id=None, alt_config=None, **kwargs):
-    try:
-        stackname = generate_stack_from_input(pname, instance_id, alt_config)
-        pdata = core.project_data_for_stackname(stackname)
-
-        print('attempting to create stack:')
```

Comment:
these (I think) were literally just temporary debug statements that I forgot to remove years and years ago. I feel like they could be expanded upon, but not here, obviously.

```
attempting to create stack 'project--instance' in 'us-east-1'. stack has:
- ec2 (t2.large)
- rds (t2.db.small, "sadfasdfsad.rds.aws.amazon.com")
- dns (instance--project.elifesciences.org)
```
etc